include/fdreadoutlibs/wib/RAWWIBTriggerPrimitiveProcessor.hpp

```diff
@@ -113,6 +130,9 @@ class RAWWIBTriggerPrimitiveProcessor
     for (size_t i = 0; i < m_nhits.size(); i++) {
       TLOG_DEBUG(1) << "Number of frames with hits " << i << ": " << m_nhits[i] << ", " << (double)((double)m_nhits[i]/(double)m_tp_frames) << "\n";
     }
+    TLOG_DEBUG(1) << "Value of wirecounter[0][0] i.e. '(wire0,fiber0)' " << m_wirecounter[0][0]; 
```

Comment:
Should these TLOGs be on a different level so they can be turned off more easily without affecting others?

Reply:
I changed the debug level to highest possible which I think is 20. There are also few debug messages at level 15.